# Generated by Django 3.1.5 on 2021-12-12 19:55

import datetime
from django.db import migrations, models
from django.utils.timezone import utc


class Migration(migrations.Migration):

    dependencies = [
        ('main', '0005_auto_20211128_1645'),
    ]

    operations = [
        migrations.RenameField(
            model_name='cliente',
            old_name='idListaPrecioDefault',
            new_name='listaPrecioDefault',
        ),
        migrations.RenameField(
            model_name='listapreciolinea',
            old_name='idListaPrecio',
            new_name='listaPrecio',
        ),
        migrations.RenameField(
            model_name='listapreciolinea',
            old_name='idProducto',
            new_name='producto',
        ),
        migrations.RenameField(
            model_name='pedidoventa',
            old_name='idListaPrecio',
            new_name='listaPrecio',
        ),
        migrations.RenameField(
            model_name='pedidoventalinea',
            old_name='idPedido',
            new_name='pedido',
        ),
        migrations.RenameField(
            model_name='pedidoventalinea',
            old_name='idProducto',
            new_name='producto',
        ),
        migrations.RenameField(
            model_name='producto',
            old_name='idFamilia',
            new_name='familia',
        ),
        migrations.RenameField(
            model_name='producto',
            old_name='idGrupo',
            new_name='grupo',
        ),
        migrations.AlterField(
            model_name='listapreciolinea',
            name='fecha_hora',
            field=models.DateTimeField(default=datetime.datetime(2021, 12, 12, 19, 54, 53, 7996, tzinfo=utc)),
        ),
        migrations.AlterField(
            model_name='pedidoventa',
            name='fecha',
            field=models.DateField(default=datetime.datetime(2021, 12, 12, 19, 54, 53, 9989, tzinfo=utc)),
        ),
    ]
